README.md

```diff
@@ -11,44 +11,46 @@ This repo contains two main programs:
 
 ## Installation
 
-pip install git+https://github.com/trompamusic/trompa-ce-client.git@tpl
+    pip install git+https://github.com/trompamusic/trompa-ce-client.git
```

Comment:
it might be better to add this as a dependency in requirements.txt instead of specifying it here